backend/src/sms/routes/sms-campaign.routes.ts

```diff
@@ -35,6 +36,11 @@ const uploadCompleteValidator = {
 
 const storeCredentialsValidator = {
   [Segments.BODY]: Joi.object({
+    label: Joi.string()
+      .min(1)
+      .max(50)
+      .pattern(/^[a-z0-9-]+$/)
+      .required(),
```

Comment:
Slight concern here to make this required. This would be a breaking change for any application that is integrating with Postman using the API.

Reply:
True, I'll assess the backward compatibility again